apscheduler/schedulers/base.py

```diff
@@ -62,12 +66,18 @@ class BaseScheduler(six.with_metaclass(ABCMeta)):
 
     .. seealso:: :ref:`scheduler-config`
     """
+    # The `group=...` API is only available in the backport, used in <=3.7, and in std>=3.10.
+    if (3, 8) <= sys.version_info <= (3, 9):
+        _trigger_plugins = {ep.name: ep for ep in entry_points()['apscheduler.triggers']}
+        _executor_plugins = {ep.name: ep for ep in entry_points()['apscheduler.executors']}
+        _jobstore_plugins = {ep.name: ep for ep in entry_points()['apscheduler.jobstores']}
+    else:
+        _trigger_plugins = {ep.name: ep for ep in entry_points(group='apscheduler.triggers')}
+        _executor_plugins = {ep.name: ep for ep in entry_points(group='apscheduler.executors')}
+        _jobstore_plugins = {ep.name: ep for ep in entry_points(group='apscheduler.jobstores')}
```

Comment:
On second thought, maybe the backport should be used for Pythons earlier than 3.10 since 3.10 is the first version where the API is stable (and `group` can be used uniformly).

Reply:
I still have a project on 3.9, and would prefer not to add an extra dependency for this somewhat minor detail. But let me know if you still want me to change it.

Reply:
Mhm...I don't have a strong opinion, given that this is a maintenance-only series. Unless a problem comes up, I won't insist.